client/src/api/okteto.ts

```diff
@@ -100,10 +101,28 @@ const up = (manifestFile: string, contextName: string, onOutputChange: (stdout:
   });
 };
 
+const status = async ( manifest: string, contextName: string): Promise<OktetoStatus> => {
+  try {
+    const args = ['status', '-f', manifest, '-c', contextName];
+    const result = await window.ddClient.extension?.host?.cli.exec(
+      'okteto',
+      args,
+    );
+    if (result) {
+      const {status} = result.parseJsonObject();
+      return status
+    }
+  } catch (_) {
+    console.error(`Error executing "okteto status" command`);
+  }
+  return 'building';
```

Comment:
Why is `building` the default status? In case of an error, I think we could add an internal status like  `unknown` or maybe the `pending` status you added in the `OktetoStatus` type.